tests/test_displacements.py

```diff
@@ -126,6 +126,22 @@ def test_plot_site_displacements_error(self):
         with pytest.raises(ValueError):
             defect_entry.plot_site_displacements(vector_to_project_on=[0, 0, 1], relative_to_defect=True)
 
+    def test_calc_displacements_ellipsoid(self):
```

Comment:
Nice! 
Could you add some quick image comparison tests (for the plotting functions, using `matplotlib`) here too, like the tests at the bottom of this module?
For this you just need to add `@custom_mpl_image_compare(filename="...")` above the test function as for the others, where `filename` is the name of the reference file to compare to in `tests/data/remote_baseline_plots`, and your test function returns the `matplotlib` `Figure` object. Would be good to do some quick tests for the different possible combinations of options.

Then maybe a quick test each like you've done here for the outputs of the function with a substitution and interstitial defect please? (Best to test each type of defect when possible). There's an example of this in `test_calc_site_displacements`